### ISSUE TITLE
ContextMenu for Word for Web no longer present

### ISSUE THREAD

**Issue by trashhalo** [OfficeDev/office-js#2872]
We have an addin in the store called Standd that describes in the manifest an DesktopFormFactor>ExtensionPoint#ContextMenu. This ContextMenu is no longer showing in Word for Web, but is showing in desktop Word (windows and mac). I found another extension called Power Thesaurus in the store that has a similar ContextMenu and theirs doesnt work either in Word for Web.

Has something happened in Word for Web to prevent all addin ContextMenus from showing up? Doesnt seem to be browser specific, happens in all browsers tested.

Unclear right place to put this problem report feel free to close and redirect me to the right place. I also filed here: https://learn.microsoft.com/en-us/answers/questions/1025522/contextmenu-for-word-for-web-missing.html

**Comment by trashhalo** [comment]
Small note we also have a ExtensionPoint PrimaryCommandSurface button on the ribbon. That IS showing up on all platforms.

**Comment by JinghuiMS** [comment]
@preethikakiru could you help answer this UX related issue. Thanks!

**Comment by trashhalo** [comment]
Confirming that context menus extension points are still not visible in word for web. I think this is a bug, or the documentation is wrong. It says this is a viable option for addins using desktopformfactor.

**Comment by dnsudha** [comment]
@preethikakiru  @trashhalo Did you get this resolved? I'm running into the same issue.

**Comment by trashhalo** [comment]
We never solved this.

**Comment by JediEagle** [comment]
Any chance anybody solved this? I have tested my add-in in browser today and I am running into the same issue. I have also been reading official documentation and it looks like this should work. On the other hand the Wikipedia addin also does not have context menu button in browser, but does in desktop version.

**Comment by kblissett** [comment]
I'm also encountering this issue. Any updates from the Word Team?

**Comment by codervishal81** [comment]
I am also facing the same issue. Custom context menu added by manifest is visible in Desktop word app. But not visible in Office 365 Word Online. Please let us know how to resolve it.

**Comment by AlexandreDewilde** [comment]
Same issue for me, I found this bug was also referenced in #3431

**Comment by sharath-spotdraft** [comment]
I am also facing the same issue. Any updates from the Word Team?

**Comment by gheorghetutunaru** [comment]
Facing the same issue. Found that on the web version some text needs to be selected for the context menu to appear after right click, but on desktop a simple right click without selecting anything works. 
Any updates about this ? Thanks

**Comment by m-hellesen** [comment]
The supported context menu tag for Word is ContextMenuText, is this the one that is not working? https://learn.microsoft.com/en-us/javascript/api/manifest/officemenu?view=common-js-preview#xsitype

**Comment by AlexandreDewilde** [comment]
Yes it's ContextMenuText that doesn't work in the browser but work in word desktop

**Comment by m-hellesen** [comment]
Thanks @AlexandreDewilde Let me look into this for you.

**Comment by m-hellesen** [comment]
@AlexandreDewilde do you happen to have a manifest you can share with me that repros this issue? That will make debugging this much faster :)
